### DISCUSSION TITLE
SimpleStorage deployment on zkSync failed

### DISCUSSION THREAD

**Discussion by triplex1** [Cyfrin/foundry-full-course-cu#2234] [Q&A] [answered]
Hello guys,

I am not sure what's wrong with the deployment, why it is always a failure. I followed everything on the video 
![image](https://github.com/user-attachments/assets/636f49eb-838b-46cc-b9fb-2c38c3cfd086)

**Comment by EngrPips** [comment]
Hello @triplex1, refresh your browser, then retry.

**Comment by cromewar** [comment]
Seems everything is fine on the screenshot you sent. Can you please confirm if this was solved? @triplex1

**Reply by triplex1** [reply]
@cromewar  Yes it's already good. Deploying and verify did not work properly, so I just uncheck it. Thanks to @EngrPips :)

**Comment by TheGrandeur** [comment, marked as answer]
use remix on incognito mode!!
